docs/insomnia/git-sync.md

```diff
@@ -6,74 +6,75 @@ category-url: get-started
 ---
 
 {:.alert .alert-primary}
-**Note**: Sync is only available to Collections that exist inside a remote Project. 
+**Note**: Sync with Git applies only to Design Documents at this time. Collections use our own Insomnia Sync for versioning. To learn about version control sync for Collections, refer to [Version Control Sync](/insomnia/version-control-sync).
 
-Insomnia is a collaborative tool for creating, managing, and sharing API specifications. This collaboration is built on the ubiquitous version control system Git, which was chosen to allow Insomnia to fit within the existing editing, review, testing, and deployment workflows that companies and teams already have in place for source code.
+Sync with Git is a built-in feature for Design Documents that enables you to configure your repository to an external Git version control system such as GitHub or BitBucket.
 
-When sync is enabled on a Collection inside a particular project, then all users that have access to the project, will also gain access to that Collection. You can manage the members in your Project from the [web dashboard](https://app.insomnia.rest/app/signup/).
+## Enable Git Sync
 
-## Enable Sync
+Enable Git Sync on local and remote Design Documents by clicking on the **Setup Git Sync** button beside Preferences. Then select **Repository Settings**. A **Configure Repository** modal will open.
```

Comment:
There's no concept of local or remote with Documents, only with Projects, which relate to Version Control Sync.

```suggestion
Enable Git Sync on Design Documents by clicking on the **Setup Git Sync** button beside Preferences. Then select **Repository Settings**. A **Configure Repository** modal will open.
```